packages/web/src/services/http/http.service.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import { api } from 'config';
+
+export const http = axios.create({
+  baseURL: api,
+})
+
+http.interceptors.request.use(
+  config => {
+    const token = localStorage.getItem('authToken');
+    console.log('token', token);
+    if (token) {
+      config.headers.authorization = `Bearer ${token}`; // eslint-disable-line no-param-reassign
```

Comment:
We should send only a token.
config.headers.authorization = token;

Reply:
Fixed